src/rt_core_v2/ids_codes/rui.py

```diff
@@ -55,11 +55,16 @@ def __eq__(self, other):
         return self.__dict__ == other.__dict__
 
 class Relationship:
-    
-    def __init__(self, uri: str, ontology: Rui):
+    """A tuple component that contains a URI for a relationship
+
+    Attributes:
+    uri -- Identifier for the temporal reference
+    """
+    def __init__(self, uri: str="http://invalid_relationship.com"):
```

Comment:
Not entirely sure here: does this mean that the default URI is the "invalid relationship" thing, just in case the user doesn't supply one?

Reply:
Yes that is the case. Its more for testing purposes and will be removed later, as it can help during testing to have something appear for the uri in either the database or json testing.